stack.yaml.lock

```diff
@@ -5,57 +5,15 @@
 
 packages:
```

Comment:
We never want to commit the `stack.yaml.lock` file so we want to revert this to how it was originally.